src/chainparams.cpp

```diff
@@ -182,6 +182,15 @@ class CMainParams : public CChainParams {
         consensus.spv.anchors_address = "1FtZwEZKknoquUb6DyQHFZ6g6oomXJYEcb";
         consensus.spv.minConfirmations = 6;
 
+        consensus.props.cfp.fee = 10 * COIN;
+        consensus.props.cfp.majorityThreshold = 5000; // vote pass with over 50% majority
+        consensus.props.voc.fee = 50 * COIN;
+        consensus.props.voc.majorityThreshold = 6667; // vote pass with over 66.67% majority
+        consensus.props.brp.fee = 500 * COIN;
+        consensus.props.brp.majorityThreshold = 6667; // vote pass with over 66.67% majority
+        consensus.props.minVoting = 1000; // 10% of the masternodes must vote
```

Comment:
We're not implementing this rule.

Reply:
Here is https://github.com/DeFiCh/ain/pull/541/commits/f39815f8d3bff7a41fcfba20cb8bdef231ce719a#diff-97c3a52bc5fad452d82670a7fd291800bae20c7bc35bb82686c2c0a4ea7b5b98R2873-R2988
On every end cycle height it's calculated active masternodes to it, then it's calculated against min voting and votes yes against all votes.